pkg/controller/volume/attachdetach/attach_detach_controller_test.go

```diff
@@ -52,7 +52,8 @@ func Test_NewAttachDetachController_Positive(t *testing.T) {
 		nil, /* prober */
 		false,
 		5*time.Second,
-		DefaultTimerConfig)
+		DefaultTimerConfig,
```

Comment:
nit: spacing

Reply:
Can you clarify?

Reply:
I think just an unnecessary change. But maybe it's a gofmt thing.